examples/requests_advanced.py

```diff
@@ -5,6 +5,12 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_requests as requests
 
+# Add a secrets.py to your filesystem that has a dictionary called secrets with "ssid" and
+# "password" keys with your WiFi credentials. DO NOT share that file or commit it into Git or other
+# source control.
+# pylint: disable=no-name-in-module,wrong-import-order
+from secrets import secrets
```

Comment:
May want to add a more verbose `try`/`except` for users who do not have a secrets.py file on their FS: https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/blob/master/examples/esp32spi_aio_post.py#L15